src/Simplex/Chat/Store/SQLite/Migrations/M20250310_group_scope.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.SQLite.Migrations.M20250310_group_scope where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+-- - group_scope_group_member_id points either to member (for chat with member as admin),
+--   or to membership (for chat with admins as member); it is used to find chat items for scope,
+--   when we know from context what group member id we are looking for;
+-- - to learn scope of chat item when context is not known, group member is joined and scope
+--   is decided based on whether member is of user member category (membership -> chat with admins), or not.
+-- TODO [knocking] TBC schema
+-- TODO            - group_members.support_chat_unanswered - don't persist, calculate on the fly?
+-- TODO            - review indexes (drop idx_chat_items_groups_item_ts?)
+m20250310_group_scope :: Query
+m20250310_group_scope =
+  [sql|
+ALTER TABLE group_members ADD COLUMN support_chat_ts TEXT;
+ALTER TABLE group_members ADD COLUMN support_chat_unanswered INTEGER;
+
+ALTER TABLE chat_items ADD COLUMN group_scope_group_member_id INTEGER REFERENCES group_members(group_member_id) ON DELETE CASCADE;
```

Comment:
the name is confusing - what suggests that it is a support chat? Also, for the chat of the current member with admins it would be Nothing?

It probably needs two fields:

group_chat_scope TEXT -- NULL - group, "support" for member support
group_chat_scope_member_id INTEGER -- NULL for member chat with admins, ID for admins' chat with member. You cannot have just one field, as the members can have both categories - their chat with admins, and their participation in support chat as admins (at a later point once promoted to admins).

Reply:
it's differentiated by joined member records (whether it's membership or member), where necessary. and when reading items list we don't need to differentiate because scope is known (and so group member id). main reason was to add single new index and not two, but as discussed proposed fields can also work with single index